snapcraft/commands/chassis.start

```diff
@@ -27,7 +28,12 @@ OVN_ARGS="--db-nb-addr="${OVN_LOCAL_IP}" \
 --db-nb-cluster-local-addr="${OVN_LOCAL_IP}" \
 --db-sb-cluster-local-addr="${OVN_LOCAL_IP}" \
 --ovn-northd-nb-db="${OVN_NB_CONNECT}" \
---ovn-northd-sb-db="${OVN_SB_CONNECT}""
+--ovn-northd-sb-db="${OVN_SB_CONNECT}" \
+--db-sb-cluster-remote-proto=ssl \
+--ovn-controller-ssl-key="${OVN_PKIDIR}"/ovn-controller-privkey.pem \
+--ovn-controller-ssl-cert="${OVN_PKIDIR}"/ovn-controller-cert.pem \
+--ovn-controller-ssl-ca-cert="${OVN_PKIDIR}"/cacert.pem"
```

Comment:
The `ovn-controller` will by default retrieve this information from the local Open vSwitch database, why not configure it there?

Reply:
The `ovn-controller` will re-read the certificate data whenever it changes on disk, so I guess this is fine.